### PR TITLE
Update split.rst

### PR DESCRIPTION
**Pull request by miguelraz**
update eltype(green) and eltype(red) to appropriate types

### CODE REVIEW

File: docs/source/split.rst [hunk 1]
```diff
@@ -119,7 +119,7 @@ The ``findwhen`` method test a condition and returns a vector of ``Date`` or ``D
     julia> green = findwhen(ohlc["Close"] .> ohlc["Open"]);
 
     julia> typeof(green)
-    Array{Date,1}
+    Vector{Date}
```
Comment (ararslan) [inline docs/source/split.rst]: This isn't correct. The type system doesn't know what a `Vector` is; it's just a type alias. Regular `Array`s always print as `Array{T,N}`.

**Comment by miguelraz** [comment]
Oh, my mistake then. 
Thanks a lot for the clarification!
Will try to be more careful next time. 

-mrg

**Comment by ararslan** [comment]
No problem. Thanks anyway for the contribution! It's always good to know that people are willing to contribute, regardless of the outcome. 🙂

**Comment by miguelraz** [comment]
Will try to contribute some more as I keep working on this.
Any and all tips/comments/suggestions are more than welcome!
Thanks for the support.

**Pull request closed without merging**